### DISCUSSION TITLE
Regarding  Mzmine3

### DISCUSSION THREAD

**Discussion by jayram1988** [mzmine/mzmine#1311] [Q&A]
Dear Mzmine3 Team,
I am trying to upload the mzML LC-MS data in the mzmine3 package, but I am getting the error message as given below:

Error parsing mzML:java.lang.NumberFormatException:For input string: "nan" (floatingDecimal.Java.2054)
Please help me in this regard.
Thanks
Jayram Bagri
NABI, India

**Comment by tomas-pluskal** [comment]
Hi Jayram,
Please file an issue here instead: https://github.com/mzmine/mzmine3/issues
Please upload the problematic mzML file and describe how you obtained it. Thanks.
